components/IssueCard.vue

```diff
@@ -0,0 +1,15 @@
+<script lang="ts" setup>
+defineProps<{ id: string; title: string }>()
+</script>
+
+<template>
+  <li
```

Comment:
The `<li>` tag should be in whichever parent is also rendering the corresponding `<ul>`.

Reply:
Changed!